src/inputs/phoneInput.jsx

```diff
@@ -13,6 +13,13 @@ const propTypes = {
     className: PropTypes.string,
     country: PropTypes.string,
     countryOptions: PropTypes.arrayOf(PropTypes.string),
+    /**
+     * A DatePickerInput can be disabled.
```

Comment:
Copy-and-paste error.  Should be "A PhoneInput can be disabled."

Reply:
Fixing...

Reply:
Commit 7c8b691 fixes this.

Reply:
Looks good.  Resolving.